packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -126,11 +129,128 @@ class AmplifyDataStorePluginTest {
         verify(mockResult, times(1)).success(emptyList<FlutterSerializedModel>())
     }
 
+    @Test
+    fun test_Delete_Success_Result_No_Predicates() {
+
+        var modelData: HashMap<String, Any> = hashMapOf(
+            "id" to "43036c6b-8044-4309-bddc-262b6c686026",
+            "title" to "Title 2",
+            "created" to "2020-02-20T20:20:20-08:00"
+        )
+
+        var instance = SerializedModel.builder()
+                .serializedData(modelData)
+                .id(modelData["id"] as String)
+                .modelName("Post")
+                .build()
+
+        var dataStoreItemChange = DataStoreItemChange.builder<SerializedModel>()
+                .item(instance)
+                .initiator(DataStoreItemChange.Initiator.LOCAL)
+                .itemClass(SerializedModel::class.java)
+                .type(DataStoreItemChange.Type.DELETE)
+                .randomUuid()
+                .build()
+
+        doAnswer { invocation: InvocationOnMock ->
+            (invocation.arguments[2] as Consumer<DataStoreItemChange<SerializedModel>>).accept(
+                    dataStoreItemChange)
+            null as Void?
+        }.`when`(mockAmplifyDataStorePlugin).delete(any(), any(QueryPredicate::class.java),
+                ArgumentMatchers.any<
+                        Consumer<DataStoreItemChange<SerializedModel>>>(),
+                ArgumentMatchers.any<Consumer<DataStoreException>>())
+
+        flutterPlugin.onDelete(mockResult,
+                readMapFromFile("delete_api",
+                        "request/instance_no_predicate.json",
+                        HashMap::class.java) as HashMap<String, Any>)
+
+        verify(mockResult, times(1)).success(
+               FlutterSerializedModel(instance).toMap())
+    }
+
+    @Test
+    fun test_Delete_Success_Result_With_Predicates() {
```

Comment:
The casing on these unit tests is weird. It's like a hybrid of `snake_case` and `camelCase`. Can you align to one or the other?

Reply:
I'll talk to @Amplifiyer about this one.

Reply:
I think I've been using snake case for test method names. I feel camel case is little difficult to read for longer sentences.

Reply:
@Amplifiyer Personally I use the `modelsCanBeSerialized()` format by habit, but I agree with you: it's harder to read when it gets long. This is basically the "Thing can be done, expressed in camel case" format.

The two other popular formats I am aware of are:
1. "Thing can be done," in snake: `models_can_be_serialized()`; or
2. `methodCall_inputState_outputState`, e.g., `serialize_validModel_correctJson()`